client/src/pages/mocks/EditMock.tsx

```diff
@@ -122,6 +122,27 @@ export const EditMock: React.FC<EditMockProps> = ({ mock, setMock }) => {
     });
   };
 
+  const onDuplicateMockResponse = (mockToDuplicate: MockResponse) => {
+    const index = mock.mockResponses.length;
+
+    return postMockResponse(snifferId as string, mockId as string, {
+      name: `${mockToDuplicate.name} copy ${index}`,
+      body: mockToDuplicate.body,
+      status: mockToDuplicate.status,
+      headers: mockToDuplicate.headers,
+      sequenceIndex: index,
+    }).then((res: MockResponse) => {
+      setMock((prev) => {
+        if (!prev) return prev;
+        return {
+          ...prev,
+          selectedResponseId: index === 0 ? res.id : prev.selectedResponseId,
```

Comment:
This line is unnecessary as the index would never be 0 in this case.

Reply:
ok

Reply:
`...prev` destruct selectedResponseId so you don't need this line.